### PR TITLE
[Tools] Enable PHPCS for data_integrity/fix_candidate_age.php

### PR DESCRIPTION
**Pull request by laemtl**

### CODE REVIEW

**Comment by pierre-p-s** [comment]
Looks like travis did not finish running for this PR somehow I dont really know why...

File: tools/data_integrity/fix_candidate_age.php [hunk 1]
```diff
@@ -71,12 +71,16 @@
 
     // Skip if Date taken does not exist
     if (!$DB->columnExists($inst, 'Date_taken')) {
-        echo "\t$inst does not use a `Date_taken` field and should be handled separately.\n";
+        echo "\t
```
Comment (driusan) [inline tools/data_integrity/fix_candidate_age.php:74]: This is changing the output (quite significantly)

File: tools/data_integrity/fix_candidate_age.php [hunk 2]
```diff
@@ -71,12 +71,16 @@
...
+        $inst does not use a `Date_taken` field and should be handled separately.
+        \n";
         continue;
     }
     // Skip if Date taken does not exist
     if (!$DB->columnExists($inst, 'Candidate_Age')) {
-        echo "\t$inst does not use a `Candidate_Age` field and should be handled separately.\n";
+        echo "\t
```
Comment (driusan) [inline tools/data_integrity/fix_candidate_age.php:81]: Same

File: tools/data_integrity/fix_candidate_age.php [hunk 3]
```diff
@@ -105,7 +114,9 @@
 
         if (!$instrument) {
             // instrument does not exist
-            echo "\t$inst for CommentID:$row[CommentID] could not be instantiated.\n";
+            echo "\t
+            $inst for CommentID:$row[CommentID] could not be instantiated.
```
Comment (driusan) [inline tools/data_integrity/fix_candidate_age.php:118]: Same

File: tools/data_integrity/fix_candidate_age.php [hunk 4]
```diff
@@ -179,7 +196,8 @@ function showHelp()
 
     echo "When the 'check' option is used, the script only detects and reports 
     miscalculated and NULL ages. 
-    Using the 'confirm' option will apply the necessary corrections to the data.\n\n";
+    Using the 'confirm' option will apply the necessary corrections to the data.
```
Comment (driusan) [inline tools/data_integrity/fix_candidate_age.php:199]: Same

**Review by driusan — CHANGES_REQUESTED** [review]

**Review by driusan — APPROVED** [review]

**Comment by laemtl** [comment]
@driusan  Ready

**Pull request merged**